src/random-walk.js

```diff
@@ -29,9 +29,9 @@ class RandomWalk {
    * @returns {void}
    */
   start (queries, period, timeout) {
```

Comment:
Please remove the lines below and change the above for using the default parameters, as follows:

```js
start (queries = ..., )
```

I know the function was not implemented by you, but feels like a good oportunity to change this.